strongbox-data-service/pom.xml

```diff
@@ -159,6 +159,10 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-aop</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax</groupId>
+            <artifactId>javaee-api</artifactId>
+        </dependency>
```

Comment:
why do we need this dependency?

Reply:
After updating the libraries, I got a compilation error: NoSuchMethodError: javax.persistence.PersistenceContext.synchronization. 
The class PersistenceContext is present in two implementations: javaee-api and hiberbate-jpa-api.
javaee-api has method synchronization, hiberbate-jpa-api has't. So I added this dependency, and excluded the other.